examples/alice_objectbox/lib/main.dart

```diff
@@ -35,12 +36,11 @@ class _MyAppState extends State<MyApp> {
   late final AliceHttpAdapter _aliceHttpAdapter = AliceHttpAdapter();
 
   late final Alice _alice = Alice(
-    showNotification: true,
-    showInspectorOnShake: true,
-    maxCallsCount: 1000,
-    aliceStorage: AliceObjectBox(
-      store: widget.store,
-      maxCallsCount: 1000,
+    configuration: AliceConfiguration(
```

Comment:
Since this is an example, declare the configuration in a separate variable, just so that it's clearer for a new user.

Reply:
👍